src/it/java/com/google/checkstyle/test/base/AbstractModuleTestSupport.java

```diff
@@ -133,7 +133,6 @@ public Checker createChecker(Configuration moduleConfig)
      * @param moduleConfig {@link Configuration} instance.
      * @return {@link Checker} instance.
      * @throws CheckstyleException if an exception occurs during checker configuration.
-     * @noinspection BooleanParameter
```

Comment:
I reopened issue 4857 to resolve item in IT area.
I ok to remove this `noinspection` if TC is green, without it.